### PR TITLE
created remember the last used folder option

### PR DESCRIPTION
**Pull request by GalievBulat**
Remember the last used folder when adding new link option

### CODE REVIEW

**Comment by GalievBulat** [comment]
Hello @AmrDeveloper I've implemented the requested feature, so please check it out

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <string name="import_export_choose_file_type">Choose a file type</string>
     <string name="password">Password</string>
     <string name="enable_password">Enable password</string>
+    <string name="default_folder">Remember the last used folder as default one</string>
```
Comment (AmrDeveloper) [inline app/src/main/res/values/strings.xml:34]: We need to find translations for other langs

File: app/src/main/java/com/amrdeveloper/linkhub/ui/setting/SettingFragment.kt [hunk 2]
```diff
@@ -55,6 +55,9 @@ class SettingFragment : Fragment() {
 
         // Setup Auto saving
         binding.autoSavingSwitch.isChecked = uiPreferences.isAutoSavingEnabled()
+
+        // Setup Auto saving
```
Comment (AmrDeveloper) [inline app/src/main/java/com/amrdeveloper/linkhub/ui/setting/SettingFragment.kt:59]: Please, Fix this comment to be `// Setup default folder`

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]
For me everything is good, just need to make two small changes

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (GalievBulat) [reply app/src/main/res/values/strings.xml:34]: So should I use google translate to translate this line to all the other languages?

Comment (AmrDeveloper) [reply app/src/main/res/values/strings.xml:34]: > So should I use google translate to translate this line to all the other languages?

For now yes, and can we make it shorter than `Remember the last used folder as default one`, hmmm maybe `Enable Remember Folder` or `Remember last folder` what do you think?

Comment (AmrDeveloper) [reply app/src/main/res/values/strings.xml:34]: Great let's go with it

**Comment by GalievBulat** [comment]
Hello, @AmrDeveloper, I've resolved the commented changes . So please check it out

**Review by AmrDeveloper — APPROVED** [review]

**Pull request merged**